ci/README.md

```diff
@@ -15,3 +15,24 @@ Individual release channels/environments are config files that are passed to inv
 ```bash
 poetry run invoke release -f channels/test.yaml
 ```
+
+## GitOps
+
+```mermaid
+flowchart TD
+classDef default fill:white, color:black, stroke:black, stroke-width:1px
+classDef group_labelstyle fill:#cde6ef, color:black, stroke-width:0px
+class ingestion_label,modeling_label,analysis_label group_labelstyle
+
+pr[Push commits to a branch.\nTest environment?]
```

Comment:
Can this say: "Does a test environment exist?". I.e., it's not a choice that you make on whether to use the test environment; if one exists, you should use it, right?